docs/design/satellite-svc-separation.md

```diff
@@ -0,0 +1,208 @@
+
+#### metainfo
+Metainfo is responsible for all things related to the metainfo stored for each file on the network. The metainfo system is currently composed of the following parts: 
+
+1) metainfoDB, a key/value store with segment path names as the key and a pointer as the value, 
+
+2) metainfo loop which iterates over all the key/values in metainfoDB, 
+
+3) metainfo endpoint which creates the public RPCs for creating/deleting metainfo, 
+
+4) metainfo service.
+
+#### orders
+Orders is responsible for creating/managing orders that the satellite issues for a file upload/download. Orders are used to keep track of how much bandwidth was used to upload/download a file. This data is used to pay storage nodes for bandwidth usage and to charge the uplinks. See this [doc on the lifecycle of data](https://github.com/storj/docs/blob/master/code/payments/Accounting.md#lifecycle-of-the-data) related to accounting which includes orders.
+
+#### audit
+Audit performs audits of the storage nodes to make sure the data they store is still retrievable. The audit system is currently made up of an audit service that runs on an interval performing audits on a segment at a time. The result of the audits are reported to the overlay service to store in node table in Satellite.DB. See [docs on audit](https://github.com/storj/docs/blob/master/code/audits/audit-service.md) for more details.
+
+#### repair
+Repair loops over the metainfoDB and checks if a segment is injured, if it is injured, its added to the repair queue and a repairer later fixes it. If its unable to be repaired then its added to irreparabledb. The irreparable loop later iterates over this table and attempts to re-repair these injured segments. The repair system is currently made of 4 parts and 2 DBs (db tables). The 4 parts are 1) repair observer (contains ReliabilityCache) 2) irreparable loop 3) repairer 4) repair queue. The 2 DBs are 1) injuredsegment (repair queue) table in satellite.DB 2) and irreparabledb table in satellite.DB 
+
+#### garbage collection (GC)
+GC consults with the metainfoDB, the source of truth for what data should be on the network, then with this info creates lists of what data each storage node should have and sends those lists to their respective storgae node.  The storage nodes can then delete any data that they are storing that is no longer in the GC list. See [GC design doc](https://github.com/storj/storj/blob/master/docs/design/garbage-collection.md) for more details. 
+
+#### accounting
+Accounting includes tally and rollup service along with the live accounting cache. The function of accounting is to calculate what to charge Uplinks for their usage and also what to pay storage nodes for their contributions to the network. See [docs on accounting](https://github.com/storj/docs/blob/master/code/payments/Accounting.md) for more details.
```

Comment:
Accounting calculates how much uplinks use storage and how much storage nodes store data.